release_checklist.sh

```diff
@@ -0,0 +1,242 @@
+#!/bin/bash
+set -euo pipefail
+
+script_path="$( cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null 2>&1 && pwd )"
+
+source ./release_utils.sh
+
+command -v gh >/dev/null || abort "Error: The Github CLI must be installed."
+
+release_type=""
+release_date=""
+version_number=""
+gb_mobile_path=""
+checklist_message=""
+issue_number=""
+include_aztec_steps=""
+include_incoming_changes=""
+debug_template=""
+auto_confirm=""
+
+while getopts "t:v:d:g:m:i:axhuy" opt; do
+  case ${opt} in
+    h )
+      echo "options:"
+      echo "   -t release type. accepts scheduled|beta|hotfix"
+      echo "   -v release version"
+      echo "   -d release date in YYYY-MM-DD format. Only used to generate scheduled release checklists"
+      echo "   -g relative path to guteberg mobile"
+      echo "   -m additional message"
+      echo "   -i existing issue to update"
+      echo "   -a include aztec steps"
+      echo "   -u include incomming steps"
+      echo "   -x echo out generated template with out sending to github"
+      echo "   -y auto confirm creating gh calls"
+      exit 0
+      ;;
+    t )
+      release_type=$OPTARG
+     ;;
+    v )
+      version_number=$OPTARG
+      ;;
+    d )
+      release_date=$OPTARG
+      ;;
+    g )
+      gb_mobile_path=$OPTARG
+      ;;
+    m )
+      checklist_message=$OPTARG
+      ;;
+    i )
+      issue_number=$OPTARG
+      ;;
+    a )
+      include_aztec_steps="true"
+      ;;
+    u )
+      include_incoming_changes="true"
+      ;;
+    x )
+      debug_template="true"
+      ;;
+    y )
+      auto_confirm="true"
+      ;;
+
+    \? )
+      echo "Invalid Option: -$OPTARG" 1>&2
+      exit 1
+      ;;
+  esac
+done
+shift $((OPTIND -1))
+
+if [[ -z "$gb_mobile_path" ]]; then
+  # Ask for path to gutenberg-mobile directory
+  # (default is sibling directory of gutenberg-mobile-release-toolkit)
+  default_gb_mobile_location="$script_path/../gutenberg-mobile"
+  read -r -p "Please enter the path to the gutenberg-mobile directory [$default_gb_mobile_location]:" gb_mobile_path
+  gb_mobile_path=${gb_mobile_path:-"$default_gb_mobile_location"}
+  echo ""
```

Comment:
I think it would be helpful to print the directory used, wdyt?

```suggestion
  echo "Using '$gb_mobile_path' as gutenberg-mobile directory"
  echo ""
```